From f0be60520d417828aad0a2689a3198fd7092f0c3 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Tue, 5 Feb 2019 15:22:06 +0100
Subject: [PATCH 6/9] popover: Disable deprecation warning

For gtk_render_frame_gap().

(cherry picked from commit c2ddb932b200dac01cc2aa03c49037f6a89fda1f)
---
 gtk/gtkpopover.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gtk/gtkpopover.c b/gtk/gtkpopover.c
index 261b9c2d1a..26f967394f 100644
--- a/gtk/gtkpopover.c
+++ b/gtk/gtkpopover.c
@@ -1184,12 +1184,14 @@ gtk_popover_draw (GtkWidget *widget,
           gap_end = final_y - rect_y;
         }
 
+G_GNUC_BEGIN_IGNORE_DEPRECATIONS
       /* Now render the frame, without the gap for the arrow tip */
       gtk_render_frame_gap (context, cr,
                             rect_x, rect_y,
                             rect_w, rect_h,
                             gap_side,
                             gap_start, gap_end);
+G_GNUC_END_IGNORE_DEPRECATIONS
     }
   else
     {
-- 
2.17.0

